locale/en/foundation/case-studies.md

```diff
@@ -5,23 +5,28 @@ layout: foundation.hbs
 
 ## Node.js Foundation Survey Results
 
-With over 3.5 million users and an annual growth rate of 100 percent, Node.js 
-is emerging as a universal platform used for web applications, IoT, and 
-enterprise. The Node.js User Survey report features insights on emerging trends 
-happening in this massive community that serves as a leading indicator on 
-trends like microservices architectures, real-time web applications, Internet 
-of Things (IoT). The report paints a detailed picture of the technologies that 
-are being used, in particular, with Node.js in production and language 
+With over 3.5 million users and an annual growth rate of 100 percent, Node.js
+is emerging as a universal platform used for web applications, IoT, and
+enterprise. The Node.js User Survey report features insights on emerging trends
+happening in this massive community that serves as a leading indicator on
+trends like microservices architectures, real-time web applications, Internet
+of Things (IoT). The report paints a detailed picture of the technologies that
+are being used, in particular, with Node.js in production and language
 preferences (current and future) for front end, back end and IoT developers.
 
 * [Read the full announcement.](/en/blog/announcements/nodejs-foundation-survey/)
 * [See the infographic.](/static/documents/2016-survey-infographic.png)
 * [Read the full report.](/static/documents/2016-survey-report.pdf)
 
 ## Node.js Drives Major Scale and Reliability at Uber
- 
-The ride-sharing platform Uber has an ambitious goal: to make transportation 
-as reliable as running water, everywhere and for everyone. Learn why Uber 
-chose Node.js to power its platform and massive-matching system. 
 
-Read the [full case study](/static/documents/casestudies/Nodejs-at-Uber.pdf).
\ No newline at end of file
+The ride-sharing platform Uber has an ambitious goal: to make transportation
+as reliable as running water, everywhere and for everyone. Learn why Uber
+chose Node.js to power its platform and massive-matching system.
+
+Read the [full case study](/static/documents/casestudies/Nodejs-at-Uber.pdf).
+
+## Node.js Helps Fusion Marketing Build Memorable Digital Experiences
+
+Fusion Marketing is a St. Louis, MO-based event marketing firm that specializes in creating custom interactive experiences. Node.js Everywhere has powered Fusion's digital expansion from developing app APIs to a 360 degree physical interactive group.
```

Comment:
Keep the limit of 80 chars per line? Mostly for consistency with the rest of the document.